spec/javascripts/components/story/expanded_story/expanded_story_state_spec.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExpandedStoryState from 'components/story/ExpandedStory/ExpandedStoryState';
+import { states } from '../../../../../app/assets/javascripts/models/beta/story';
+
+describe('<ExpandedStoryState />', () => {
+  it('renders component title', () => {
+    const onEditSpy = sinon.spy();
+
+    const story = { state: 'started', _editing: { state: 'started' } };
+
+    const wrapper = shallow(<ExpandedStoryState story={story} onEdit={onEditSpy} />);
+
+    expect(wrapper.text()).toContain(I18n.t('activerecord.attributes.story.state'));
+  });
+
+  describe("states at select", () => {
+    states.map(state => {
```

Comment:
You can use `states.forEach` here instead of `states.map` since the return of `.map()` here is not being used 😄